monitor.js

```diff
@@ -159,43 +233,60 @@ const Monitor = module.exports = {
 			}
 		}
 	},
-	writeNetworkUse: function () {
+
+	writeNetworkUse() {
 		let buf = '';
 		for (let i in this.networkUse) {
-			buf += '' + this.networkUse[i] + '\t' + this.networkCount[i] + '\t' + i + '\n';
+			buf += `${this.networkUse[i]}\t${this.networkCount[i]}\t${i}\n`;
 		}
 		FS('logs/networkuse.tsv').write(buf);
 	},
-	clearNetworkUse: function () {
+
+	clearNetworkUse() {
 		if (Config.emergency) {
 			this.networkUse = {};
 			this.networkCount = {};
 		}
 	},
+
 	/**
 	 * Counts roughly the size of an object to have an idea of the server load.
+	 *
+	 * @param {any} object
+	 * @return {number}
 	 */
-	sizeOfObject: function (object) {
-		let objectList = [];
+	sizeOfObject(object) {
+		/** @type {Set<(Array | Object)>} */
+		let objectCache = new Set();
 		let stack = [object];
 		let bytes = 0;
 
 		while (stack.length) {
 			let value = stack.pop();
-			if (typeof value === 'boolean') {
+			switch (typeof value) {
+			case 'boolean':
 				bytes += 4;
-			} else if (typeof value === 'string') {
+				break;
+			case 'string':
 				bytes += value.length * 2;
-			} else if (typeof value === 'number') {
+				break;
+			case 'number':
 				bytes += 8;
-			} else if (typeof value === 'object' && !objectList.includes(value)) {
-				objectList.push(value);
-				for (let i in value) stack.push(value[i]);
+				break;
+			case 'object':
+				if (!objectCache.has(value)) objectCache.add(value);
+				if (Array.isArray(value)) {
+					for (let el of value) stack.push(el);
+				} else {
+					for (let i in value) stack.push(value[i]);
+				}
+				break;
 			}
 		}
 
 		return bytes;
 	},
-};
 
-Monitor.cleanInterval = setInterval(() => Monitor.clean(), 2 * 60 * 60 * 1000);
```

Comment:
Mainly I wanted to remove any references to `global.Monitor` in the module, since it technically isn't defined until after the module has finished loading and `app.js` has assigned it.